src/lib/fragments/Accordion.react.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+
+import { defaultProps, propTypes } from '../components/ComponentTemplate.react';
```

Comment:
Should we update this to import props from our new accordian component file (Accordian.react.js)?